scripts/hcnmgr

```diff
@@ -40,7 +41,7 @@ PHYSLOC=""
 DEVPATH=""
 VIO_TYPE=""
 DISTRO=""
-SERVICES=""
+SERVICE=""
 
 source /usr/sbins/functions.suse
```

Comment:
This should be `source /usr/sbin/functions.suse` as I guess, not **`sbins`**, but it's not yet in Makefile.am or spec file.
But I'd rather install it to `%_libexecdir/%{name}/` than to `%_sbindir/`...